tests.edn

```diff
@@ -0,0 +1,16 @@
+#kaocha/v1 {:tests [{:id :unit
+                     :ns-patterns ["rems"]
+                     :test-paths ["src" "test"]
+                     :skip-meta [:browser :integration]}
+                    {:id :browser
+                     :ns-patterns ["rems"]
+                     :test-paths ["src" "test"]
+                     :focus-meta [:browser]}
+                    {:id :integration
+                     :ns-patterns ["rems"]
+                     :test-paths ["src" "test"]
+                     :focus-meta [:integration]}]
```

Comment:
this is different from the previous behaviour of integration

Reply:
It is, because by default (without any flags) kaocha runs all test suites, and thus we would end up running unit tests twice, in two suites.

We need to either then stop using kaocha without any flags and always specify just one suite, give multiple suites to kaocha or use lein aliases.

Reply:
oh right, kaocha uses suites and not selectors so these kinda need to be mutually exclusive